SPECS/kernel/config_aarch64

```diff
@@ -7575,6 +7575,7 @@ CONFIG_HID_ZYDACRON=m
 CONFIG_HID_SENSOR_HUB=m
 CONFIG_HID_SENSOR_CUSTOM_SENSOR=m
 CONFIG_HID_ALPS=m
+# CONFIG_HID_MCP2200 is not set
```

Comment:
Result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/b289b83409a6edd262840a6bac4d990e1ef87971